sklearn/svm/tests/test_svm.py

```diff
@@ -655,6 +655,38 @@ def test_linearsvc_crammer_singer():
     assert_array_almost_equal(dec_func, cs_clf.decision_function(iris.data))
 
 
+def test_linearsvc_fit_sampleweight():
+    # check correct result when sample_weight is 1
+    n_samples = len(X)
+    unit_weight = np.ones(n_samples)
+    clf = svm.LinearSVC(random_state=0).fit(X, Y)
+    clf_unitweight = svm.LinearSVC(random_state=0).\
+        fit(X, Y, sample_weight=unit_weight)
+
+    # check if same as sample_weight=None
+    assert_array_equal(clf_unitweight.predict(T), clf.predict(T))
+    assert_allclose(np.linalg.norm(clf.coef_),
+                    np.linalg.norm(clf_unitweight.coef_), 1, 0.0001)
```

Comment:
Bare with me, but why are you comparing the norms, and not the coef_ themselves? Maybe I missed something.